src/thumbnail.c

```diff
@@ -793,7 +793,13 @@ void feh_thumbnail_show_fullsize(feh_file *thumbfile)
 			winwidget_show(thumbwin);
 	} else if (FEH_FILE(thumbwin->file->data) != thumbfile) {
 		thumbwin->file = l;
+#ifdef HAVE_INOTIFY
+        winwidget_inotify_remove(thumbwin);
+#endif
 		feh_reload_image(thumbwin, 1, 1);
```

Comment:
Same as winwidget.c: force_new=0 seems to make more sense here.

```suggestion
		feh_reload_image(thumbwin, 1, 0);
```